.github/actions/sdk-codegen/entrypoint.sh

```diff
@@ -49,6 +60,16 @@ function generate_go {
   TEMPLATE_DIR=$GENERATOR_DIR/go_templates
   export GH_REPO=$GO_SDK_REPO
   git clone https://github.com/algorand/go-algorand-sdk $GO_SDK_DIR
+  
+  # Clean previously (stale) generated files before regenerating them.
+  # Some files are hand-written instead of generated, so preserve those files here.
+  find $GO_SDK_DIR/client/v2/common/models/* $GO_SDK_DIR/client/v2/algod/* $GO_SDK_DIR/client/v2/indexer/* \
```

Comment:
The same rules for Go apply to the JS SDK?

Reply:
No, each SDK has unique rules. I didn't invent these, just imported from each predefined script, e.g. https://github.com/algorand/generator/blob/88668a4cabad2a7c4ca6f8426c1c55b091231939/scripts/generate_go.sh#L66-L73